db/versions/0023.yml

```diff
@@ -0,0 +1,138 @@
+version: 23
+migrationScript: 0023-migration.sql
+downgradeScript: 0023-downgrade.sql
+methods:
+  roles_entities_load:
+    description: See taskcluster-lib-entities
+    mode: read
+    serviceName: auth
+    args: partition_key text, row_key text
+    returns: table (partition_key_out text, row_key_out text, value jsonb, version integer, etag uuid)
+    body: |-
+      begin
+        -- if no roles, return canned "empty" value, as the below expression will return NULL
+        perform 1 from roles limit 1;
+        if not found then
+          return query
+          select
+            'role',
```

Comment:
Should these have a named columns? (e.g., `'name' as partition_key`)

Reply:
I *think* this is the case where postgres ignores the aliases, so it's easier to leave them off than to include them and potentially have a mismatch when something is added or removed.  I'll double-check.

Reply:
Confirmed, adding `as foo` to one of these didn't cause any errors, so postgres is only looking at the order of the `select`ed columns.